README.md: Add Quick install section

jre-wine opened the pull request:
For https://github.com/Winetricks/winetricks/issues/1192.

austin987 commented:
Lgtm, but maybe should include instructions to install it in /usr/local/bin? That way it's in $PATH and doesn't require invoking with sh /full/path

qwertychouskie commented:
Honestly the quick install script should be just:
```bash
sudo curl -o /usr/local/bin/winetricks https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
sudo chmod +x /usr/local/bin/winetricks
```
And the full install:
```bash
sudo curl --create-dirs -o /usr/local/bin/winetricks https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
sudo chmod +x /usr/local/bin/winetricks
sudo curl --create-dirs -o /usr/local/share/man/man1/winetricks.1 https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.1
sudo curl --create-dirs -o /usr/local/share/applications/winetricks.desktop https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.desktop
sudo curl --create-dirs -o /usr/local/share/metainfo/winetricks.appdata.xml https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.appdata.xml
sudo curl --create-dirs -o /usr/local/share/icons/hicolor/scalable/apps/winetricks.svg https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.svg
sudo curl --create-dirs -o /usr/local/share/bash-completion/completions/winetricks https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.bash-completion
```
Much simpler and more complete than the old instructions.  However, curl seems to be creating the directories with `drwxr-x---` as the permissions (no read for standard users), maybe one of you knows a way around this that won't complicate things a bunch.

Actually, even better, add a `--self-install`  option to Winetricks, then the entire install would be:
```
wget -qO - https://github.com/Winetricks/winetricks/raw/master/src/winetricks | pkexec sh /dev/stdin --self-install
```
The `--self-install` function would `mkdir -p` the necessary directories, then download the files to their locations as above.  `self-install` would default to `/usr/local` as the prefix, while e.g. `self-install=/usr` would use `/usr` as the prefix.
```
qwerty@qwerty-Inspiron-3520:~$ wget -qO - https://github.com/Winetricks/winetricks/raw/master/src/winetricks | pkexec sh /dev/stdin --self-install
------------------------------------------------------
unknown option --self-install
------------------------------------------------------
qwerty@qwerty-Inspiron-3520:~$ 
```
Let's make that option known! :)

melroy89 commented on README.md:
````diff
@@ -26,19 +26,32 @@ The ```winetricks``` package should be used if it is available and up to date. T
+
+You can then run it from a terminal with
+~~~~~
+sh winetricks
````
I will not advise it like this. I would first use chmod and just execute script as intended..

```suggestion
# Make executable
chmod +x ./winetricks

# Run
./winetricks
```

jre-wine replied:
See new comment below. But thanks @melroy89

melroy89 commented on README.md:
````diff
@@ -26,22 +26,32 @@ The ```winetricks``` package should be used if it is available and up to date. T
+
+To quickly grab current winetricks open a terminal and type
+~~~~~
+wget  https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
````
Ps. I never seen `~~~~` but I think ``` is normally used for a code block.

Like so:

```sh
echo "Hello world";
````

Or python highlighting:

```py
print("hello world");
```

jre-wine replied:
Thanks, I fixed the formatting.

There's an "Outdated" label in your comment. Not sure what that implies.

melroy89 replied:
That means, if you are unfamiliar with GitHub PRs, you committed a change this this file. Making the current code/markdown I see above outdated. Hence the reason GitHub marks it now as outdated. 

Ps. Which doesn't mean the code is actually fixed correctly, again only saying there is a newer version.

jre-wine replied:
Ah, thanks. So I marked this comment as resolved.

jre-wine commented:
Updated and added "-O winetricks" to overwrite a previously downloaded winetricks. Otherwise the new one would get a suffix and the user would execute the old one.

I'm open to both changing this to /usr/local/bin/ and/or chmod +x. Above is just my reasoning for doing it otherwise, but each solution qualifies as a "Quick install", so I have no hard opinion on this.
